net/netfilter/nft_set_rbtree.c

```diff
@@ -464,58 +466,80 @@ static void nft_rbtree_walk(const struct nft_ctx *ctx,
 
 static void nft_rbtree_gc(struct work_struct *work)
 {
-	struct nft_rbtree_elem *rbe, *rbe_end = NULL, *rbe_prev = NULL;
-	struct nft_set_gc_batch *gcb = NULL;
+	struct nft_rbtree_elem *rbe, *rbe_end = NULL;
+	struct nftables_pernet *nft_net;
 	struct nft_rbtree *priv;
+	struct nft_trans_gc *gc;
 	struct rb_node *node;
 	struct nft_set *set;
+	unsigned int gc_seq;
+	struct net *net;
```

Comment:
This is actually a part of this commit that applied cleanly during the rebuild as a part of `kernel-4.18.0-534.el8`

We might want to make it a preconditional patch just so the history is as accurate as possible ... its techincally in the `netfilter:` list but its `netfilter: nft_set_rbtree`.

I was hoping we would have to reach out of `nf_tables` but it appears that somet things are kinda interwoven.

Reply:
I guess technically this commit overwrites all the changes from the above commits ... and I would think should have marked the the usptream commit `commit https://github.com/ctrliq/kernel-src-tree/commit/5d235d6ce75c12a7fdee375eb211e4116f7ab01b` as fixed in this commit but they didn't.

So I think we have two options.
1. Integrate `netfilter: nft_set_rbtree: skip elements in transaction from garbage collection` https://github.com/ctrliq/kernel-src-tree/commit/5d235d6ce75c12a7fdee375eb211e4116f7ab01b
2. In `Upstream-diff` we comment that `5d235d6ce75c12a7fdee375eb211e4116f7ab01b` is completely overwritten by this commit so we're not backporting it.

Reply:
In this case I'm going say that my preference is not to pull in a piece of code in one patch series that is overwritten by another patch in the same series.  We're documenting the history well enough by adding your suggested comment for upstream-diff.

Commit message updated in the next upcoming push.